src/bin/server/accounts/account.rs

```diff
@@ -7,8 +7,8 @@ pub enum Error {
     Underflow,
 }
 
-/// Contains the balance for a user
-#[derive(Debug)]
+/// Contains the balance and last processed sequence for a user
```

Comment:
```suggestion
/// Contains the balance and the latest processed sequence for a user
```

We hope there is another one...